packages/pds/src/db/message-queue/index.ts

```diff
@@ -37,8 +39,11 @@ export class SqlMessageQueue implements MessageQueue {
     }))
 
     await tx.db.insertInto('message_queue').values(values).execute()
-
-    this.processNext()
+    for (let i = 0; i < msgArray.length; i++) {
+      this.processNext().catch((err) => {
```

Comment:
Right now this initiates a handful of queries which will end-up pending.  Each one might take-up a connection from the connection pool while they wait.  It's a minor thing, but if we serialize those ourselves rather than leaning on the connection pool and db locks, it could be fruitful.  That might look like `this.processNext(n)`, and allowing `processNext()` to internally loop `n` times.

Reply:
took this to chat, but just for posterity's sake: we're nearly always doing one event at a time here, max 2. so this isn't the big bottleneck. 

we'll keep an eye on the message queue tho once we get into connection pool scaling issues